### PR TITLE
Don't auto-update on tagged builds, only snapshots

### PR DESCRIPTION
**Pull request by kipz**

### CODE REVIEW

File: src/main/scripts/travis-build.bash [hunk 1]
```diff
@@ -44,7 +44,7 @@ function main() {
             return 1
         fi
         if ! wget https://api-staging.atomist.services/model/schema -O src/main/resources/com/atomist/rug/ts/cortex.json; then
-            err "Failed to download production cortex json"
+            err "Failed to download staging cortex json"
```
Comment (ddgenome) [inline src/main/scripts/travis-build.bash]: Should we always download staging? Perhaps on release we should be downloading the production version?

File: src/main/scripts/npm-publish.bash [hunk 2]
```diff
@@ -37,29 +38,45 @@ function publish() {
...
-        return 1
-    fi
+    # npm honors this
+    rm -f "$target/.gitignore"
 
-    if [[ $NPM_TOKEN ]]; then
-        msg "Creating local .npmrc using API key from environment"
-        if ! ( umask 077 && echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > "$HOME/.npmrc" ); then
-            err "failed to create $HOME/.npmrc"
+    if [[ $TRAVIS_TAG =~ ^[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
```
Comment (ddgenome) [inline src/main/scripts/npm-publish.bash]: Should we key off the `$module_version` instead of the `$TRAVIS_TAG`?

**Review by ddgenome — COMMENTED** [review]
Thanks for consolidating the scripts.

File: src/main/scripts/npm-publish.bash (hunk 2 above)
Comment (kipz) [reply src/main/scripts/npm-publish.bash]: makes sense.

File: src/main/scripts/travis-build.bash (hunk 1 above)
Comment (kipz) [reply src/main/scripts/travis-build.bash]: Yeah - that _is_ what we are doing - unless I missed something?

Comment (ddgenome) [reply src/main/scripts/travis-build.bash]: You're right, I just didn't expand the diff to see the above one.

**Comment by ddgenome** [comment]
I've done some things in the POM I'm not proud of, but this should work for both release and non-release builds as you wanted @kipz . If you are OK with the changes, go ahead and merge.

File: pom.xml [hunk 3]
```diff
@@ -71,7 +71,7 @@
                             <goal>install-node-and-npm</goal>
                         </goals>
                         <configuration>
-                            <nodeVersion>v7.2.1</nodeVersion>
+                            <nodeVersion>v6.10.2</nodeVersion>
```
Comment (kipz) [inline pom.xml:74]: good catch, though we may want to upgrade!

**Comment by kipz** [comment]
Thanks @ddgenome

**Pull request merged**